util/chplenv/chpl_llvm.py

```diff
@@ -972,24 +965,33 @@ def use_flag(flag_to_use):
             return True
         return False
 
+    skip_sysroot = False
+
     # if the user set one of the flags, use it
     flag_to_use = _determine_gcc_flag_to_use()
     if flag_to_use:
         use_flag(flag_to_use)
+    elif (any(arg.startswith("--sysroot") for arg in clang_cfg_args) or
+        any(arg.startswith("--gcc-install-dir") for arg in clang_cfg_args) or
+        any(arg.startswith("--gcc-toolchain") for arg in clang_cfg_args)):
+        # Check that the clang configure file doesn't already supply
+        # info about the sysroot or gcc install so we don't end up overriding it.
```

Comment:
I find this comment confusingly worded. How can setting `skip_sysroot = True` check anything? Is the idea that this is about the conditional above? If so, reword to "If the clang configure file ..."

Reply:
Yes this comment was about the if statement above. I've updated the comment to better reflect its nesting